src/System.ServiceModel.Shim/System.ServiceModel.Shim.csproj

```diff
@@ -7,7 +7,6 @@
     <GenFacadesIgnoreMissingTypes>true</GenFacadesIgnoreMissingTypes>
     <!--Don't produce or publish symbols for Facade assemblies.-->
     <PublishWindowsPdb>false</PublishWindowsPdb>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
```

Comment:
Why did you remove this?  I'm missing where this project sets this property now.

Reply:
Does it actually need that property?
For this project we don't want to use any of the logic in FacadeAssemblies.targets right?
GenFacades and ResolvedMatchingContract happens right there in the same project.

By leaving IsPartialFacadeAssembly set to true it was also running the ResolveMatchingContract target in FacadeAssemblies.targets and trying to find a related Ref.csproj.

Reply:
My above comments are not completely correct.
By removing  `<IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>` I was also not triggering the AddReferenceAssemblyToPackage target in FacadeAssemblies.targets
Made changes to call that target and added a condition to skip the logic in the ResolveMatchingContract target.

@ericstj I do see one diff from previous packages that I'm not sure about. Maybe we can follow-up on this next week, the shim shipped in 4.5.3 also had a reference to System.Runtime that I am not getting right now. Not sure if it is needed.

Reply:
I believe that GenFacades requires the property in order to run the facade generation target.  Did you confirm it still has type forwards after this change?

Reply:
My last update before merging added that property back and made a couple additional modifications to make sure we were doing the right thing regarding the targets in FacadeAssemblies.targets over the weekend our vendors did some package testing including specifically the shim and everything looked good.